x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/components/install_command_utils.test.ts

```diff
@@ -17,7 +17,7 @@ describe('getInstallCommandForPlatform', () => {
       );
 
       expect(res).toMatchInlineSnapshot(`
```

Comment:
Should we add a test case for this scenario?

Reply:
Added here https://github.com/elastic/kibana/pull/120549/commits/226c94c2edf5057dd200987653d43bcc57ff4d63